### PR TITLE
return aliases for desc coll

### PR DESCRIPTION
**Pull request by lsgrep**
#4812 
- return aliases when describing a collection.

### CODE REVIEW

File: internal/rootcoord/root_coord_test.go [hunk 1]
```diff
@@ -1822,9 +1839,9 @@ func TestRootCoord(t *testing.T) {
 	req2 := &milvuspb.CreateCollectionRequest{
 		Base: &commonpb.MsgBase{
 			MsgType:   commonpb.MsgType_CreateCollection,
-			MsgID:     3013,
-			Timestamp: 3013,
-			SourceID:  3013,
+			MsgID:     3014,
```
Comment (yhmo) [inline internal/rootcoord/root_coord_test.go:1842]: Why change 3013 to 3014, 3014 to 3015, etc?

Comment (lsgrep) [reply internal/rootcoord/root_coord_test.go:1842]: To keep the Timestamp ordering intact in the UTs.

**Comment by scsven** [comment]
Hi @lsgrep, commit message should start with a capital letter, and be clear and meaningful, for example, describe what changes you made.

**Pull request merged**